server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -762,7 +763,13 @@ public void apply(Settings value, Settings current, Settings previous) {
                 // Composite index settings
                 CompositeIndexSettings.STAR_TREE_INDEX_ENABLED_SETTING,
 
-                SystemTemplatesService.SETTING_APPLICATION_BASED_CONFIGURATION_TEMPLATES_ENABLED
+                SystemTemplatesService.SETTING_APPLICATION_BASED_CONFIGURATION_TEMPLATES_ENABLED,
+
+                // QueryGroup settings
```

Comment:
We are missing `QUERY_GROUP_RUN_INTERVAL_SETTING`.

Reply:
After discussion, we decided to remove `QUERY_GROUP_RUN_INTERVAL_SETTING` as it'll be defined in the service itself